### DISCUSSION TITLE
my Database Function return only 1000 rows.

### DISCUSSION THREAD

**Discussion by tangible-idea** [supabase/supabase#26770] [Questions] [answered]
When I select my data manually, it shows over 6600 rows.
but when I tried to use Function which is supposed to get the same result, it got only 1000 rows.

```
response = supabase.rpc('get_records_in_time_range3', {'start_timestamp': time_72_hours_ago,'end_timestamp': time_end, 'room_id': roomTag}).execute()
    print("data len ====> " +str(len(response.data)))
```
This shows `data len ====> 1000`

<img width="676" alt="Screenshot 2024-05-26 at 18 51 58" src="https://github.com/supabase/supabase/assets/4496005/6056566e-8988-40fb-9911-c46fe180354a">

<img width="584" alt="Screenshot 2024-05-26 at 18 52 48" src="https://github.com/supabase/supabase/assets/4496005/1d8f5f34-d4e2-486f-bd99-0426e55789a0">

**Comment by GaryAustin1** [comment, marked as answer]
There is a dashboard API setting that limits the rows.  Default is 1000.
![image](https://github.com/supabase/supabase/assets/54564956/2cb86491-f06d-4fb8-8bf7-bbb65d889385)

**Comment by tangible-idea** [comment]
@GaryAustin1 That works for me!
